### PR TITLE
feat: added different varients of card component

### PR DESCRIPTION
**Pull request by Manju-sree**
Here is the Card Component pull request for below variants in cards.

- Cards with badges
- Cards with dismiss
- Cards with text overlay
- Text only cards
- Vertical/Horizontal cards
- Cards with shadow

### CODE REVIEW

File: Components/Card/card.css [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+        margin-left: 5rem;
+    }
+    
+    .title-align-center {
+        margin-top: 2rem;
+    }
+    /* card with badge */
+    
+    .card-vertical {
+        max-width: 208px;
```
Comment (Raghav888) [inline Components/Card/card.css]: Please use rem

File: Components/Card/card.css [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+        width: 100%;
+        position: relative;
+        box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 5%);
+        margin: 1rem 1rem 0rem 1rem;
+    }
+    
+    .image-container {
+        width: 200px;
```
Comment (Raghav888) [inline Components/Card/card.css]: use rem only

**Review by Raghav888 — APPROVED** [review]
Please use rem, its a standard practice

File: Components/Card/card.css [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+    
+    .badge-container {
+        position: relative;
+        width: fit-content;
+        box-sizing: border-box;
+    }
+    
+    .card-container .image-container {
+        margin: 0.1rem 0;
+    }
```
Comment (sks-21) [inline Components/Card/card.css:35]: For values lesser than 10px you should prefer using px rather than rems.

File: Components/Card/card.css [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+    
+    .card-container,
+    .card-img {
+        border-radius: 0.3rem;
+    }
```
Comment (sks-21) [inline Components/Card/card.css:40]: same as above

**Review by sks-21 — APPROVED** [review]
Rest everything looks good

File: Components/Card/card.css (hunk 1 above)
Comment (Manju-sree) [reply Components/Card/card.css]: done changes

File: Components/Card/card.css (hunk 2 above)
Comment (Manju-sree) [reply Components/Card/card.css]: done changes here also

File: Components/Card/card.css (hunk 4 above)
Comment (Manju-sree) [reply Components/Card/card.css:40]: yeah, will try to implement from next commits

**Pull request merged**